config/locales/find.yml

```diff
@@ -151,6 +151,10 @@ en:
       primary_title: Visa sponsorship
       secondary_title: Do you need visa sponsorship?
     search:
+      primary: Primary (ages 3 to 11)
+      browse_primary: Browse primary courses
+      browse_secondary: Browse secondary courses
+      secondary: Secondary (ages 11 to 18)
```

Comment:
Small question: is the idea on everything homepage translation to be centralised?